content/docs/04-clusters/03-edge/04-edgeforge-workflow/03.5-palette-canvos.md

```diff
@@ -0,0 +1,507 @@
+
+  Refer to the Advanced workflow in the current guide to learn how to use another registry, Docker Hub, and tag the docker images accordingly.
+
+  Use the following commands to push the the provider OS image to the *ttl.sh* image registry.  
+  <br />
+
+  ```bash
+  docker push ttl.sh/ubuntu:k3s-1.25.2-v3.3.3-palette-learn
+  ```
+
+  ```bash
+  docker push ttl.sh/ubuntu:k3s-1.24.6-v3.3.3-palette-learn
+  ```
+  <br />
+
+  <WarningBox>
+
+  As a reminder, *ttl.sh* is a short-lived image registry. If you do not use these provider OS images in your cluster profile within 24 hours of pushing to *ttl.sh*, they will expire and must be re-pushed.
+
+  </WarningBox>
+  <br />
+
+
+# Validate
+1. List the edge installer ISO image and checksum by issuing the following command from the **CanvOS/** directory.
+<br />
+
+  ```shell
+  ls build/
+  ```
+
+  ```shell
+  # Output
+  palette-edge-installer.iso      
+  palette-edge-installer.iso.sha256
+  ```
+  <br />
+
+2. List the Docker images to show two provider OS images - one compatible with K3s v1.24.6 and another with K3s v1.25.2.
+<br />
+
+  ```shell
+  docker images
+  ```
+
+  ```bash coloredLines=3-4
+  # Output
+  REPOSITORY        TAG                                IMAGE ID       CREATED         SIZE
+  ttl.sh/ubuntu     k3s-1.25.2-v3.3.3-palette-learn    b3c4956ccc0a   6 minutes ago   2.49GB
+  ttl.sh/ubuntu     k3s-1.24.6-v3.3.3-palette-learn    fe1486da25df   6 minutes ago   2.49GB
+  earthly/earthly   v0.7.4                             d771cc8edc38   2 weeks ago     333MB
+  ```
+
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Advanced" key="advanced_create_artifacts">
+
+1. Check out the [CanvOS](https://github.com/spectrocloud/CanvOS.git) GitHub repository containing the starter code. 
+  <br />
+
+  ```bash
+  git clone https://github.com/spectrocloud/CanvOS.git
+  ```
+  
+
+2. Change to the **CanvOS/** directory. 
+  <br />
+
+  ```bash
+  cd CanvOS
+  ```
+
+
+3. The files relevant to the current guide are in the **v3.3.3** tag. Check out the desired tag. You can check out a more recent and stable version if available.
+  <br />
+
+  ```shell
+  git checkout v3.3.3
+  ```
+
+
+4. Review the relevant files.  
+  <br />
+
+  ```bash
+  .
+  ├── .arg          # Defines mutiple variables
+  ├── Dockerfile    # Bakes the arguments to use in the image
+  ├── Earthfile     # Series of commands for creating target images
+  ├── earthly.sh    # Script to invoke the Earthfile, and generate target images
+  └── user-data.template  # A sample user-data file
+  ```
+  <br />
+
+
+5. Edit the **.arg** file to customize the image tag for provider OS images, image registry, image repository, and OS distribution. 
```

Comment:
Step five is way to large. Multiple steps are happening below. Break those out to its own number.